src/scrabble.js

```diff
@@ -1,5 +1,33 @@
 class Scrabble {
   // Write your implementation here
+constructor(word) {
+this.word = word
+}
+
+  
```

Comment:
Maybe you want to add an object in here to be able to iterate through it to get the scores for each letter